# This message carries all stuff necessary to draw things on the captured image
Header header
sensor_msgs/Image frame
geometry_msgs/Point32[] rh_2D_landmarks
geometry_msgs/Point32[] lh_2D_landmarks
geometry_msgs/Point32[] pose_2D_landmarks